packages/e2e.test.foundry-sdk-generator/src/legacy-client/search.test.ts

```diff
@@ -446,10 +447,12 @@ describe("SearchObjects", () => {
     const employees = assertOkOrThrow(result);
     expect(employees.data.length).toEqual(1);
     expect(employees.data[0].fullName).toEqual("John Doe");
-    expectTypeOf(employees.data[0]).toEqualTypeOf<{
+    expectTypeOf(employees.data[0]).branded.toEqualTypeOf<{
       readonly fullName: string | undefined;
       readonly __primaryKey: number;
       readonly __apiName: "Employee";
+      readonly $primaryKey: number;
```

Comment:
The e2e tests in `foundry-sdk-generator` were not being typechecked before. For some reason they were excluded from the tsconfig. Thus when we did things like add `$primaryKey` to the output, we didn't know we broke tests. I didn't exclude them after the move so I had to fix the tests too